lib/test_track.rb

```diff
@@ -57,8 +57,4 @@ def private_url
   def enabled?
     enabled_override.nil? ? !Rails.env.test? : enabled_override
   end
-
-  def fully_qualified_cookie_domain_enabled?
```

Comment:
This seemed very specific to the session cookie management, so I moved it into a private method in session.rb.